CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contribution
+
+Please read [Auth0's contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md).
+
+## Documentation
+
+- PR for docs site update, if needed
+- Code-level documentation expectations
+	- 100% documentation coverage for PRs
+	- Include links to relevant Auth0 doc pages
+
+## Assets compilation
+
+Information about compiling CSS, JS, SVG, etc.
```

Comment:
Is this a TODO? Seems like dummy content. I'd say remove it if we're not adding anything helpful here.